bdsx/bds/packets.ts

```diff
@@ -674,16 +674,22 @@ export class CameraPacket extends Packet {
 export class BossEventPacket extends Packet {
     @nativeField(bin64_t)
     unknown:bin64_t;
-    @nativeField(bin64_t)
-    entityUniqueId:bin64_t;
```

Comment:
Please keep the legacy for backward compatibility.
need to reduce the damage to old scripts.

Reply:
Ok I will rename it back and add a description on the field